### PR TITLE
add inverses

### PR DESCRIPTION
**Pull request by aplavin**
Convenient in eg `Accessors.@modify(..., deopcompose(func))`.

Only 1.9, no change on earlier julias.

### CODE REVIEW

File: ext/InverseFunctionsCompositionsBaseExt.jl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module InverseFunctionsCompositionsBaseExt
```
Comment (jw3126) [inline ext/InverseFunctionsCompositionsBaseExt.jl]:
```suggestion
module CompositionsBaseInverseFunctionsExt
```
I think package first is the more standard convention?

Comment (aplavin) [reply ext/InverseFunctionsCompositionsBaseExt.jl]: Totally!

**Comment by aplavin** [comment]
Register please :) This PR bumped the version for convenience.

**Comment by aplavin** [comment]
@jw3126 could you register the new version please?

**Comment by jw3126** [comment]
sorry yeah!

**Pull request merged**